docs/modules/operation/pages/search/search.adoc

```diff
@@ -0,0 +1,86 @@
+
+== Search view
+
+The Search view provides a number of search options for nodes.
+Click *Search* on the main menu to access the *Search* view.
+
+=== Quick search links
+
+Click on the following links to quickly search for the associated information:
+
+* All nodes
+* All nodes and their interfaces
+* All nodes with asset info
+
+=== Name containing search
+
+Search by name is a case-insensitive, inclusive search.
+For example, searching on `serv` finds any of `serv, Service, Reserved, NTSERV, UserVortex`, and so on.
+Use the underscore character as a single-character wildcard.
+Use the percent character as a multiple-character wildcard.
+
+=== TCP/IP Address like search
+
+Searching by TCP/IP address uses a flexible search format that lets you separate the four or eight (in the case of IPv6) fields of a TCP/IP address into specific searches.
+An asterisk (*) in place of any octet matches any value for that octet.
+Separate two numbers by a dash (-) to indicate a range.
+Use a comma for list demarcation.
+
+For example, the following search fields are all valid and each create the same result set: all TCP/IP addresses from 192.168.0.0 through 192.168.255.255:
+
+* 192.168.\*.*
+* 192.168.0-255.0-255
+* 192.168.0,1,2,3-255.*
+* 2001:6a8:3c80:8000-8fff:*:*:*:*
+* fc00,fe80:*:*:*:*:*:*:*
+
+=== System or interface attribute search
+
+A System/Interface attribute search finds nodes with systems or interfaces that match a given search string.
+The "contains" modifier provides a case-insensitive, inclusive search similar to the "name containing" search described above.
+The "equals" modifier requires the search to find an exact match.
+
+=== Location or Providing service search
+
+Select the location or service to search for from the appropriate drop-down menu.
+
+=== MAC address like search
+Case-insensitive, partial string match to find interfaces with hardware (MAC) addresses that match the search string.
+For example, to find all interfaces with a specified manufacturer's code, enter the first six characters of the MAC address.
+Octet separators (dash or colon) are optional.
+
+=== Foreign Service name like search
+Case-insensitive, inclusive search on the foreign service name.
+Use the underscore character as a single-character wildcard.
+Use the percent character as a multiple-character wildcard.
+
+=== Flows search
+Find nodes with or without flows data.
+
+=== Enhanced Linkd topology search
+
+Find nodes with CDP/LLDP data that match the given search string.
+
+=== Category or Field asset information search
+Search for nodes that have assets associated with a particular category or field.
+The field search includes a list of fields and a text box to match a value.
+The field search is case-insensitive and inclusive.
+Use the underscore character as a single-character wildcard.
+Use the percent character as a multiple-character wildcard.
```

Comment:
Probably should also include info about customizing that was part of a recent release.
Per `opennms.properties`:
```
# By default the central search shows only the found nodes and the corresponding matched field. This property
# allows you to add an additional line where you can use any meta-data DSL expression.
#
# This example will display the node's sys-contact and the associated department:
# org.opennms.netmgt.search.info=${node:sys-contact} (Department: ${asset:department})
#
# By default this property is not used and empty.
# org.opennms.netmgt.search.info=
```

```suggestion
Use the percent character as a multiple-character wildcard.

== Customizing search results

It is possible to customize the search results to show additional metadata in the search results pop up.
By creating a new file in `$OPENNMS_HOME/etc/opennms.properties.d/` and specifying a value for `org.opennms.netmgt.search.info`, you can pick additional metadata fileds to display.

This example will display the node's sys-contact and the associated department:

 org.opennms.netmgt.search.info=${node:sys-contact} (Department: ${asset:department})